docs/en/FAQ/CatFindNewModule.md

```diff
@@ -0,0 +1,22 @@
+### Problem
+when you want add a new module in SW, you will encounter a error which is
```

Comment:
1. The first letter should b upper case in every paragraph. 
1. `SW` -> `SkyWalking`
1. `will` -> `may`
1. `a error` -> `the following error`
1. Remove `which is`.

Reply:
OK, i got it. thanks.